src/formatDate.js

```diff
@@ -8,7 +8,31 @@
  * @returns {string}
  */
 function formatDate(date, fromFormat, toFormat) {
-  // write code here
+  const dateParts = date.split(fromFormat[3]);
+  const oldVersion = {};
+  const newDate = [];
+
+  for (let i = 0; i < 3; i++) {
+    oldVersion[fromFormat[i]] = dateParts[i];
+
+    if (fromFormat[i] === 'YYYY' && toFormat.includes('YY')) {
+      oldVersion['YY'] = oldVersion['YYYY'].slice(2);
+    }
```

Comment:
When converting from 'YYYY' to 'YY', it's not necessary to check if 'YY' is included in the 'toFormat'. Since the 'fromFormat' already includes 'YYYY', you can directly convert it to 'YY'. This will simplify the logic and make the code more efficient.